driver/gl/window.go

```diff
@@ -488,26 +488,28 @@ func (w *window) findObjectAtPositionMatching(canvas *glCanvas, mouse fyne.Posit
 	if canvas.overlay != nil {
 		content = canvas.overlay
 	}
-	canvas.walkObjects(content, fyne.NewPos(0, 0), false, func(walked fyne.CanvasObject, pos fyne.Position) {
+	canvas.walkObjects(content, fyne.NewPos(0, 0), func(walked fyne.CanvasObject, pos fyne.Position) bool {
 		if !walked.Visible() {
-			return
+			return false
 		}
 
 		if mouse.X < pos.X || mouse.Y < pos.Y {
-			return
+			return false
 		}
 
 		x2 := pos.X + walked.Size().Width
 		y2 := pos.Y + walked.Size().Height
 		if mouse.X >= x2 || mouse.Y >= y2 {
-			return
+			return false
 		}
 
 		if matches(walked) {
 			found = walked
 			foundX, foundY = mouse.X-pos.X, mouse.Y-pos.Y
+			return true
```

Comment:
This return true may not be correct... We actually need to find the deepest object in a tree that matches the criteria (such as a Tappable object within a Tappable object, or Scrollable within Scrollable). Which means sometimes we must pass a found object to found a more relevant result.

Reply:
Oh … I will try to create a test case but I'm not sure that I will succeed. However, the `true` will be changed into `false`.